app/helpers/meta_data_helper.rb

```diff
@@ -5,7 +5,7 @@ module MetaDataHelper
   include ActionView::Helpers::TagHelper
 
   def og_prefix
-    'og: http://ogp.me/ns# article: http://ogp.me/ns/article# profile: http://ogp.me/ns/profile#'
+    'og: https://ogp.me/ns# article: https://ogp.me/ns/article# profile: https://ogp.me/ns/profile#'
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

Reply:
https://github.com/diaspora/diaspora/issues/8198#issuecomment-775280343

> * HTML namespaces like the opengraph example: No need - those URLs are for spec-purposes only, they will not get queried by a browser.

Reply:
These URLs provided the reason for this PR and Issue.
BTW, the ogp URLs are prefixes for the og: Protocols. In their documentation, every URL is a https - URL, so even this is not a technical need, changing (as many as possible) these URLs modernize the Diaspora output. Don't show a skeptical user, that the diaspora devs don't care about secure protocols. 

I herby declare a request to change this.

Reply:
> These URLs provided the reason for this PR and Issue.

That still doesn't mean it makes any sense. ;)

They changed their docs [some time in July last year](http://web.archive.org/web/20200702012303/https://ogp.me/) and didn't even note it somewhere because it has absolutely no relevance whatsoever.

Nobody cares about namespace URLs. They will not show up in your browser, they will not show up as a mixed content warning, they won't even show up in the devtools. The only reasonable response to someone looking at the HTML source and asking why there are http links is: "That absolutely has no relevance whatsoever.". I mean, what's next? People complaining that all our svg's have `xmlns="http://www.w3.org/2000/svg"` and `http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd` as their dtd?! Like, c'mon.

But fine, I don't care enough to spend any more time on discussing this. If you want to change it, sure.